scripts/ci/jobs/check-generated.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/env bash
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"/../../.. && pwd)"
+source "$ROOT/scripts/ci/lib.sh"
+
+set -euo pipefail
+
+go mod tidy
+
+# shellcheck disable=SC2016
+echo 'Ensure that generated files are up to date. (If this fails, run `make proto-generated-srcs && make go-generated-srcs` and commit the result.)'
+function generated_files-are-up-to-date() {
+    ln -s /go/src/github.com/stackrox /go/src/github.com/rox
```

Comment:
I don't get how this works?! How does creating a softlink like this solve the refs to `github.com/stackrox/rox`:
```
# ls -l /go/src/github.com/
total 0
lrwxrwxrwx. 1 root root 27 Jun 27 21:02 rox -> /go/src/github.com/stackrox
drwxr-xr-x. 4 root root 22 Jun 27 21:00 stackrox
```